test/interop/fortran/FortranCallChapel/chapelProcs.prediff

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+FTN=gfortran
+
+$FTN -c testCallChapel.f90 -fno-underscoring
+$FTN testCallChapel.o -Llib -lchapelProcs `$CHPL_HOME/util/config/compileline --libraries` -o chapelProcs -fno-underscoring
```

Comment:
I'm a little surprised you're using the make clean command but not the other commands you wrote in the makefile.  Is that intentional?

Reply:
Good point, I can just "make" instead of writing out the full gfortran commands.